### ISSUE TITLE
bug(a11y): add more precise labels for next and previous day buttons

### ISSUE THREAD

**Issue by jeripeierSBB** [lyne-design-system/lyne-components#1898]
`sbb-datepicker-next-day` and `sbb-datepicker-previous-day` should have aria-labels like "Change to the next day, currently selected %s" to better indicate what happens.

specs:
 - "%" should be replaced by current date
 - If the field is today, the current date should instead be "Today"
 - If the field is empty, the prev/next buttons should be disabled

**translations next day:**
- de : "Zum nächsten Tag wechseln, derzeit ausgewählt %s"
- en:  "Change to the next day, currently selected %s"
- fr:  "Passer au jour suivant, actuellement sélectionné  %s"
- it:  "Passare al giorno successivo, attualmente selezionato %s"

**translations previous day:**
- en: "Change to the previous day, currently selected %s"
- de: "Zum vorherigen Tag wechseln, derzeit ausgewählt %s"
- it: "Passare al giorno precedente, attualmente selezionato %s"
- fr: "Passer au jour précédent, actuellement sélectionné %s"

**translations today:**
- en: "today"
- de: "heute"
- fr: "aujourd'hui"
- it: "oggi"
